narwhals/_expression_parsing.py

```diff
@@ -95,7 +94,7 @@ def evaluate_into_exprs(
     """Evaluate each expr into Series."""
     series: ListOfCompliantSeries = [  # type: ignore[assignment]
         item
-        for sublist in (evaluate_into_expr(df, into_expr) for into_expr in flatten(exprs))
+        for sublist in (evaluate_into_expr(df, into_expr) for into_expr in exprs)
```

Comment:
ooh nice - was this never needed then?

Reply:
`*expr` should always be a tuple. Not sure why we flatten it?!